Source/Client/Sound/AudioPlaybackEngine.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using NAudio.Wave;
+using NAudio.Wave.SampleProviders;
+
+namespace FireAndForgetAudioSample
+{
+    class AudioPlaybackEngine : IDisposable
+    {
+        private readonly IWavePlayer outputDevice;
+        private readonly MixingSampleProvider mixer;
+
+        public AudioPlaybackEngine(int sampleRate = 44100, int channelCount = 2)
+        {
+            outputDevice = new WaveOutEvent();
+            mixer = new MixingSampleProvider(WaveFormat.CreateIeeeFloatWaveFormat(sampleRate, channelCount));
+            mixer.ReadFully = true;
+            outputDevice.Init(mixer);
+            outputDevice.Play();
+        }
+
+        public void PlaySound(string fileName)
```

Comment:
So far, this is only a sample code. Could you please add some calls to it somewhere in the actual game code?